python/kserve/kserve/model_server.py

```diff
@@ -99,6 +116,7 @@ def start(self, models: Union[List[Model], Dict[str, Deployment]], nest_asyncio:
             for model in models:
                 if isinstance(model, Model):
                     self.register_model(model)
+                    self.validate_log_latency(model)
```

Comment:
Should this be done in the `__init__`  of ModelServer?

Reply:
changed.